plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryThumbnails.php

```diff
@@ -120,7 +120,7 @@ protected function should_display_view_all( $thumbnails_count, $product_gallery_
 	 * @return string Rendered block type output.
 	 */
 	protected function render( $attributes, $content, $block ) {
```

Comment:
Do you think we should early return an empty string if there are no gallery images? E.g.

```
if ( ! $product_gallery_images || count( $product_gallery_images ) <= 1 ) {
    return '';
}
```

Reply:
Good catch!